vyper/opcodes.py

```diff
@@ -149,6 +149,7 @@
     'CLAMP_NONZERO': [None, 1, 1, 19],
     'ASSERT': [None, 1, 0, 85],
     'ASSERT_REASON': [None, 3, 0, 85],
+    'ASSERT_UNREACHABLE': [None, 1, 0, 85],
```

Comment:
Hm, how is the gas estimate calculated for this?

Reply:
Usualy calculated by hand, will review